ovgs.proto

```diff
@@ -254,14 +257,17 @@ service OwnershipVoucherService {
   // AddUserRole will assign a role to a user in a named group.
   // Username is unique to an Org ID and Username.
   // Errors will be returned:
+  // INVALID_ARGUMENT if the username is empty
+  // NOT_FOUND if the group doesn't exist, as specified in the request
   // FAILED_PRECONDITION if any of the paramters doesn't exist.
-  // ALREADY_EXISTS if user already exists in the group with the role.
```

Comment:
'with the role' was removed. I'm not sure if it makes sense to have the same user have multiple roles on the same group. What set of permissions do they actually inherit? The lowest ones? It seems easier to just enforce a single role per user  per group.